### PR TITLE
change not started exception message and const name

### PR DESCRIPTION
**Pull request by yosriz**

### CODE REVIEW

File: kin-devplatform-sdk/src/main/java/kin/devplatform/exception/ClientException.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 	public static final int INTERNAL_INCONSISTENCY = 4003;
 	public static final int ORDER_NOT_FOUND = 4004;
 	public static final int ACCOUNT_NOT_LOGGED_IN = 4005;
+	public static final int SDK_IS_NOT_STARTED = 4005;
```
Comment (drEfrati) [inline kin-devplatform-sdk/src/main/java/kin/devplatform/exception/ClientException.java]: Did you intentionally made it the same number as ACCOUNT_NOT_LOGGED_IN(4005)?
If not then change, if yes then just merge

File: .circleci/config.yml [hunk 2]
```diff
@@ -14,6 +14,8 @@ jobs:
     
     steps:
       - checkout
+      - run: yes | sdkmanager --licenses || exit 0
+      - run: yes | sdkmanager --update || exit 0
```
Comment (OrenZak) [inline .circleci/config.yml]: 🥇

File: kin-devplatform-sdk/src/main/java/kin/devplatform/exception/ClientException.java (hunk 1 above)
Comment (yosriz) [reply kin-devplatform-sdk/src/main/java/kin/devplatform/exception/ClientException.java]: yes, kind of backward compatibility if someone outside using this const (internally I'm using the new one)

File: .circleci/config.yml (hunk 2 above)
Comment (yosriz) [reply .circleci/config.yml]: :rose:

**Pull request merged**